src/lucky/routable.cr

```diff
@@ -58,6 +58,21 @@ module Lucky::Routable
     end
   {% end %}
 
+  # Define a route that responds to a WebSocket request
+  macro ws(path, &block)
+    {% unless path.starts_with?("/") %}
+      {% path.raise "Path must start with a slash. Example: '/#{path}'" %}
+    {% end %}
+
+    {% if block.args.size != 1 %}
+      {% raise "ws takes a block with 1 arg." %}
+    {% end %}
+
+    add_route({{method}}, {{ path }}, {{ @type.name.id }})
```

Comment:
We will either need to have a separate router for websockets or do something to separate them from http routes. I don't think `{{ method }}` is a thing? We could make that "ws" or something I think?

Reply:
Yeah, that should be `:ws` there. 😂